modules/javafx.graphics/src/main/native-font/fontpath_linux.c

```diff
@@ -410,9 +410,13 @@ Java_com_sun_javafx_font_FontConfigManager_getFontConfig
 
             fontformat = NULL;
             (*FcPatternGetString)(fontPattern, FC_FONTFORMAT, 0, &fontformat);
-            /* We only want TrueType fonts for Java FX */
-            if (fontformat != NULL
-                && (strcmp((char*)fontformat, "TrueType") != 0)) {
+            /* We only want OpenType fonts for Java FX :
+             * ie TrueType and CFF format fonts.
+             */
+            if ((fontformat != NULL) &&
+                ((strcmp((char*)fontformat, "TrueType") != 0) &&
```

Comment:
Code in `javax.web` uses `equalLettersIgnoringASCIICase` in a similar situation, should we do the same here?

Reply:
I thought about that but we have been using specific-case here for ever for TrueType and also in the Java 2D code - which BTW does allow CFF already and it looks for 'CFF' and more to the point the fontconfig API we are using itself explicitly looks
for 'CFF' which it in turn gets by using freetype to examine font files and the API in freetype it uses says
===
   *  FT_Get_Font_Format
   *
   * @description:
   *  Return a string describing the format of a given face.  Possible values
   *  are 'TrueType', 'Type~1', 'BDF', 'PCF', 'Type~42', 'CID~Type~1', 'CFF',
   *  'PFR', and 'Windows~FNT'.
 ===

So it is baked in from the start what case is used.